### PR TITLE
fullscreen-mode-video-size

### PR DESCRIPTION
**Pull request by sxian0613**
In full-screen mode, adjust the size of the video according to whether the timeline and bottom control bar should be displayed on the screen.

### CODE REVIEW

File: packages/web-components/src/player-component/player-component.style.ts [hunk 1]
```diff
@@ -278,10 +278,30 @@ export const styles = css`
...
+
+    :host(.live-off.timeline-off.bottom-controls-off) .fullscreen video {
+        max-height: calc(100%);
+    }
+
+    :host(.live-on.bottom-controls-on) .fullscreen video {
+        max-height: calc(100% - ${bottomControlHeight});
+    }
+
+    :host(.live-on.bottom-controls-off) .fullscreen video {
```
Comment (giakas) [inline packages/web-components/src/player-component/player-component.style.ts:301]: May be we should group similar rules together using comma separation rules

Comment (sxian0613) [reply packages/web-components/src/player-component/player-component.style.ts:301]: Sure, I will do that. 
And I will also do that and update the code in PR #100.

Comment (sxian0613) [reply packages/web-components/src/player-component/player-component.style.ts:295]: The similar rules are grouped together.

**Comment by sxian0613** [comment]
The changes are included in PR #100.

**Pull request closed without merging**